client/components/download-link/renderers/docx-renderer.js

```diff
@@ -361,7 +368,14 @@ export default (application, sections, values, updateImageDimensions) => {
     }
     const nodes = content.document.nodes;
 
-    nodes.forEach(node => renderNode(doc, node));
+    nodes.forEach(node => {
+      try {
+        renderNode(doc, node);
+      } catch (e) {
+        doc.createParagraph(`There was a problem rendering this content (${node.type})`).style('error');
+        doc.createParagraph(e.stack).style('error');
```

Comment:
is this going to dump the stack trace in production?

Reply:
Yes.

Given that a) it's super rare that these error, b) there's no actual logging done so this is likely our best shot at debugging, and c) this is a public repo anyway I'm not totally bothered by that fact.